README.md

```diff
@@ -3,7 +3,7 @@
 # Git Extensions
 
 Git Extensions is a standalone UI tool for managing git repositories.
-It also integrates with Windows Explorer and Microsoft Visual Studio (2015/2017).
+It also integrates with Windows Explorer and Microsoft Visual Studio (2015/2017/2019).
```

Comment:
The same should probably be copied into the project description, too:
![image](https://user-images.githubusercontent.com/46861028/58289787-b626e400-7db7-11e9-8e92-523d3e946e99.png)

Reply:
There is at least #6500 before VS2019 is fully supported....
README should be updated

Reply:
As I understand, #6500 is for building GE itself with VS2019.
But is anything blocking the use of the current VSIX extension in VS2019?

Reply:
No